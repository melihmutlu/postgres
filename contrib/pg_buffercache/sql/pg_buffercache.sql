CREATE EXTENSION pg_buffercache;

select count(*) = (select setting::bigint
                   from pg_settings
                   where name = 'shared_buffers')
from pg_buffercache;

select used_buffers + unused_buffers > 0,
        dirty_buffers < used_buffers,
        pinned_buffers < used_buffers
from pg_buffercache_summary();