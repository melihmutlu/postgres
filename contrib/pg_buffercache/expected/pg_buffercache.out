CREATE EXTENSION pg_buffercache;
select count(*) = (select setting::bigint
                   from pg_settings
                   where name = 'shared_buffers')
from pg_buffercache;
 ?column? 
----------
 t
(1 row)

select used_buffers + unused_buffers > 0,
        dirty_buffers < used_buffers,
        pinned_buffers < used_buffers
from pg_buffercache_summary();
 ?column? | ?column? | ?column? 
----------+----------+----------
 t        | t        | t
(1 row)

